src/tests/JIT/CodeGenBringUpTests/CastThenBinop_d.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You have 4 csproj here, but for this test I'd use just a single one with the following content:
```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <DebugType>None</DebugType>
    <Optimize>True</Optimize>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="CastThenBinop.cs" />
  </ItemGroup>
</Project>
```

Reply:
acf28b8